### DISCUSSION TITLE
How to Populate ON-Screen Text with a Variable Value

### DISCUSSION THREAD

**Discussion by brucerothwell** [WarmUpTill/SceneSwitcher#1253] [Q&A]
If I have a text object on screen, and I set a variable to be a psecific value. how can I make the text object display the value of the variable?

**Comment by WarmUpTill** [comment]
You can use the "Source" action type to change the settings of text source and modify the "Text" property.
To assign the value of a variable to this property use [this syntax](https://github.com/WarmUpTill/SceneSwitcher/wiki/Variables#using-variables): `${variable name here}`.

Here is an example:
![Example](https://github.com/user-attachments/assets/1939362c-7e37-4b0f-955b-b37e2cfb9909)

Let me know if you have further questions!

**Comment by brucerothwell** [comment]
Excellent!Beautiful!Sent from my iPhoneOn Oct 31, 2024, at 9:50 AM, WarmUpTill ***@***.***> wrote:﻿
You can use the "Source" action type to change the settings of text source and modify the "Text" property.
To assign the value of a variable to this property use this syntax: ${variable name here}.
Here is an example:
Example.gif (view on web)
Let me know if you have further questions!
